ddtrace/internal/datadog/profiling/cmake/FindCppcheck.cmake

```diff
@@ -0,0 +1,8 @@
+include(ExternalProject)
+
+ExternalProject_Add(cppcheck_project
+  GIT_REPOSITORY https://github.com/danmar/cppcheck.git
+  GIT_TAG "2.13.3"
+  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}/cppcheck
```

Comment:
That's correct.  Ordinarily, I'd shove a prebuilt cppcheck into a docker container or pull it from the repo, but so far I had managed to avoid adding additional dependencies, and even though this isn't incorporated into CI yet, it's nice not to depend too much on my own system's setup.

I'm not 100% sure how this will get handled in the sequel.  There are a few other things happening around splitting dd-trace-py which may make this easier.